cypress.json

```diff
@@ -8,6 +8,7 @@
   "defaultCommandTimeout": 10000,
   "videoUploadOnPasses": false,
   "chromeWebSecurity": false,
+  "blockHosts": ["www.google-analytics.com", "www.googletagmanager.com"],
```

Comment:
honey badger too? and is walkme a thing, or is that just loaded from google tag manager?

Reply:
what is url for honeybadger?